components/profileName/profileName.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { View, Text, StyleSheet, ScrollView } from 'react-native';
+
+const ProfileName = () => {
+    return (
+        <ScrollView style={styles.ScrollView}>
+            <View style={styles.body}>
+                <Text style={styles.titleText}>Максим Чернявский</Text>
+            </View>
+        </ScrollView>
+    );
+};
+
+const styles = StyleSheet.create({
+    titleText: {
+        fontSize: 30,
+        fontWeight: "bold",
+        color: "white",
+        textAlign: 'center',
+        fontFamily: "Cochin",
+        marginTop: 20,
+    },
+    ScrollView: {
+        backgroundColor: '#5e29e3',
```

Comment:
such things like color should be in main Color file. In root level of folders you need to create 'res' folder, where you will have colors.js file with map object, that will have all colors that you use.
`
export default {
white: 'white',
purple: '#5e29e3'
};
`